tutorials/tree/dictionary/writeTree.cpp

```diff
@@ -0,0 +1,61 @@
+// ##########################################################################
+// # Alvaro Tolosa Delgado @ IFIC(Valencia,Spain)  alvaro.tolosa@ific.uv.es #
+// # Copyright (c) 2018 Alvaro Tolosa. All rights reserved.		 #
+// ##########################################################################
+
+#ifndef __writeTree_cpp__
+#define __writeTree_cpp__
+
+#include <iostream>
+
+#include <TFile.h>
+#include <TTree.h>
+#include <TTreeReader.h>
+#include <TTreeReaderValue.h>
+
+#include "data2Tree.hpp"
+
+void writeTree()
+{
+
+   TFile *ofile = TFile::Open("testFile.root", "recreate");
```

Comment:
I would use a `std::unique_ptr` instead here (see https://root.cern/manual/trees/).  I think it's more inline with modern C++ guidelines.